tools/run_tests/distribtest_targets.py

```diff
@@ -97,7 +97,14 @@ def build_jobspec(self):
           ['test/distrib/csharp/run_distrib_test.sh'],
           environ={'EXTERNAL_GIT_ROOT': '../../..'})
     else:
```

Comment:
I don't see much value in doing that (considering we know we currently support only linux mac an windows, and in this form the script will surely fail if not run on windows). I'm fine with making the change if you insist though.